src/components/common/RecipientsTable/RecipientTable.js

```diff
@@ -106,20 +107,20 @@ const RecipientTable = ({
   useEffect(() => {
     getAllRecipientAction();
     getAllHouseholdAction();
+    console.log('CURRENT_RECIPIENT: ', recipientToEdit);
```

Comment:
I don't think it is great practice to have console.logs in production, but I might be wrong about that.